pkg/controller/controller.go

```diff
@@ -522,3 +522,27 @@ func isInstanceReady(instance *v1alpha1.Instance) bool {
 
 	return false
 }
+
+func isOwnedByBinding(objectMeta *metav1.ObjectMeta, binding *v1alpha1.Binding) bool {
+	ref := getControllerRef(objectMeta)
+	if ref == nil {
+		return false
+	}
+	return ref.APIVersion == binding.APIVersion &&
+		ref.Kind == binding.Kind &&
+		ref.Name == binding.Name &&
```

Comment:
@nilebox, I believe it's how it works for any resource kinds. See https://github.com/kubernetes/client-go/issues/60#issuecomment-281747911
I encountered this issue many times but feel free to double check.